src/Hazelcast.Net.Tests/Resources/Cluster/cp.xml

```diff
@@ -43,8 +43,7 @@
 
   <cp-subsystem>
     <base-dir>/custom-cp-dir</base-dir>
-    <cp-member-count>3</cp-member-count>
-    <group-size>3</group-size>
```

Comment:
why removed?

Reply:
The default group size is already equals to cp member count at default.